lighthouse-core/audits/first-meaningful-paint.js

```diff
@@ -40,6 +40,7 @@ class FirstMeaningfulPaint extends Audit {
       name: 'first-meaningful-paint',
       description: 'First meaningful paint',
       optimalValue: SCORING_POINT_OF_DIMINISHING_RETURNS.toLocaleString() + 'ms',
+      helpText: 'First meaningful paint measures when the primary content of a page appears to be visible. <a href="https://developers.google.com/web/tools/lighthouse/audits/first-meaningful-paint" target="_blank" rel="noreferrer noopener">Learn more</a>.',
```

Comment:
> appears to be visible

If something appears it's definitely visible :P

Not sure of the exact wording, but maybe "appears on screen"? Unless you mean, this is our best guess that the primary content is visible?